condor/execute.sh

```diff
@@ -36,6 +36,12 @@ for key in workflow output samplejson year campaign isSyst isArray noHist overwr
     ARGS[$key]=$(jq -r ".$key" $WORKDIR/arguments.json)
 done
 
+# Create base env with python=3.10 and setuptools<=70.1.1
+micromamba activate 
+micromamba install python=3.10 -c conda-forge xrootd --yes
```

Comment:
Do we really need to specify xrootd here? I think we can get rid of this?

Reply:
> Do we really need to specify xrootd here? I think we can get rid of this?

I would need to test it

Reply:
Removing the installation of xrootd in the script throws the following error:
```bash
  File "/pool/condor/dir_1916719/micromamba/lib/python3.10/site-packages/uproot/extras.py", line 78, in XRootD_client
    raise ModuleNotFoundError(
ModuleNotFoundError: Install XRootD python bindings with:

    conda install -c conda-forge xrootd
```

Reply:
Thanks for checking that!